src/metadata/countries.json

```diff
@@ -2120,8 +2120,50 @@
   [
     "ae",
     "United Arab Emirates",
-    "971",
-    "+971 (..) ... ... ...."
+    "9712",
+    "+971 (.) ... ...."
+  ],
+  [
+    "ae",
+    "United Arab Emirates",
+    "9713",
+    "+971 (.) ... ...."
+  ],
+  [
+    "ae",
+    "United Arab Emirates",
+    "9714",
+    "+971 (.) ... ...."
+  ],
+  [
+    "ae",
+    "United Arab Emirates",
+    "9715",
+    "+971 (..) ... ...."
+  ],
+  [
+    "ae",
+    "United Arab Emirates",
+    "9716",
+    "+971 (.) ... ...."
+  ],
+  [
+    "ae",
+    "United Arab Emirates",
+    "9717",
+    "+971 (.) ... ...."
+  ],
+  [
+    "ae",
+    "United Arab Emirates",
+    "9718",
+    "+971 (.) ... ...."
+  ],
+  [
+    "ae",
+    "United Arab Emirates",
+    "9719",
+    "+971 (.) ... ...."
```

Comment:
```suggestion
    "971",
    "+971 (.) ... ... ....."
  ],
  [
    "ae",
    "United Arab Emirates",
    "9715",
    "+971 (5.) ... ... ...."
```

Reply:
@ArtyomVancyan Thanks for the quick review, I far as my research UAE phone number is 7 digit. This is the mail issue I am trying to address with this PR.

Reply:
Please check these examples https://github.com/google/libphonenumber/tree/31a5a03f27421250409fdc03e41a8d7b9af3d655/resources/metadata/971

Reply:
In the directory you have referenced above, there is an [examples.csv](https://github.com/google/libphonenumber/blob/31a5a03f27421250409fdc03e41a8d7b9af3d655/resources/metadata/971/examples.csv) where numbers have 9 digit length. You can create an issue on their repository by pointing it out and referencing the Wiki page.